### PR TITLE
gencffi: Support bool types.

### PR DESCRIPTION
**Pull request by corona10**
gencffi: Support bool types.

### CODE REVIEW

**Comment by corona10** [comment]
@sbinet PTAL

File: _examples/simple/simple.go [hunk 1]
```diff
@@ -12,3 +12,10 @@ func Func() {}
 func Add(i, j int) int {
 	return i + j
 }
+
+func Bool(b bool) bool {
```
Comment (sbinet) [inline _examples/simple/simple.go]: it's a bit of a confusing test function.
why not just:

```go
func Bool(b bool) bool {
   return b
}
```

or perhaps:
```go
func Equal(a, b int) bool {
   return a == b
}
```

(or did you want to also test `bool`eans as arguments of functions? in that case I would stick with my former `Bool()` function)

Comment (corona10) [inline _examples/simple/simple.go]: > or did you want to also test booleans as arguments of functions?

Yeah, That's what I intended to do.

**Comment by corona10** [comment]
Updated!

**Pull request merged**